05_deploy_bootstrap_vm.sh

```diff
@@ -143,4 +143,27 @@ ssh -o "StrictHostKeyChecking=no" core@$IP sudo podman run \
 # Create a master_nodes.json file
 jq '.nodes[0:3] | {nodes: .}' "${WORKING_DIR}/ironic_nodes.json" | tee ocp/master_nodes.json
 
+MASTER_INTERFACE="eth1"
+
+# Fix etcd discovery on bootstrap
+rm -rf ocp/machineconfigs
+mkdir -p ocp/machineconfigs/temp
+# Find master machine config name
+while [ -z $(ssh -o StrictHostKeyChecking=no "core@$IP" sudo ls /etc/mcs/bootstrap/machine-configs/master*) ]; do sleep 5; done
+
+MASTER_CONFIG=$(ssh -o StrictHostKeyChecking=no "core@$IP" sudo ls /etc/mcs/bootstrap/machine-configs/master*)
```

Comment:
observation: the `master*` glob could match more than one file, noting just in case.

Reply:
There are two predefined groups of assets installer generates - master and worker. These are hardcoded and won't change anytime soon, could be renamed to 'control_plane' though.

MCS adds a hash (possibly a checksum?) in the file name, so full file name can't be specified here.